modules/release-notes/pages/relnotes.adoc

```diff
@@ -65,55 +113,80 @@ SELECT ... FROM ... WHERE ...
 ----
 |===
 
-*Data Service*
+==== Data Service
 
-[#table_knownissues_v55-kv-data,cols="25,66"]
+[#table_knownissues_v600-kv-data,cols="25,66"]
 |===
 | Issue | Description
 
+| https://issues.couchbase.com/browse/MB-30074[MB-30074^]
+| *Summary*: Following a Memcached crash, the data node is observed to be stuck in a pending state.
+
 | https://issues.couchbase.com/browse/MB-29809[MB-29809^]
-| *Summary*: The `last_modified` can be incorrect if a bucket was upgraded and contains old documents.
+| *Summary*: The `last_modified` attribute can be incorrect if a bucket was upgraded and contains old documents.
```

Comment:
Also need to add release notes for https://issues.couchbase.com/browse/MB-31410:

*Summary*: If bucket memory usage exceeds high watermark and reaches 99% of bucket quota, a race condition in KV-Engine DCP consumer is exposed which may result in DCP messages being processed out-of-order.

Reply:
Added